### ISSUE TITLE
l2_loss

### ISSUE THREAD

**Issue by hkhatod** [thtrieu/qclass_dl#3]
Why are you added the output of l2_loss(b) to l2_loss(self.w) ? It didnt make sense to me. Could you please explain what am I missing ?

  l2_loss += tf.nn.l2_loss(self.W)
  l2_loss += tf.nn.l2_loss(b)

Thanks.

**Comment by thtrieu** [comment]
This is L2 weight decay, a form of regularization.
